nf_test4/Makefile.am

```diff
@@ -13,8 +13,13 @@ LDADD = ${top_builddir}/fortran/libnetcdff.la
 
 # These programs test the F77 netCDF-4 API using include netcdf.inc.
 NF77_TESTS = ftst_groups ftst_vars ftst_vars2 ftst_vars3 ftst_vars4	\
-ftst_vars5 ftst_vars6 ftst_vars7 ftst_types ftst_types2 ftst_types3	\
```

Comment:
Do we need to make corresponding changes to the `CMakeLists.txt` file for the changes in this file, as well?